ruby/gps_pub.rb

```diff
@@ -0,0 +1,7 @@
+require 'ably'
+
+lat, long = 10.1233, 12.22
+
+ably = Ably::Realtime.new(key: ENV['ABLY_API_KEY'])
+channel = ably.channels.get 'driver:elon'
```

Comment:
This style doesn't align with our Ruby style in our SDK.  This should use parenthesis, `ably.channels.get('driver:elon')`. See https://github.com/ably/ably-ruby

Reply:
Fixed